opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -166,116 +168,35 @@ BlackoilPropsAdFromDeck::BlackoilPropsAdFromDeck(const BlackoilPropsAdFromDeck&
 
         phase_usage_ = phaseUsageFromDeck(deck);
 
+        gasPvt_ = std::make_shared<GasPvt>();
+        oilPvt_ = std::make_shared<OilPvt>();
+        waterPvt_ = std::make_shared<WaterPvt>();
 
+        gasPvt_->initFromDeck(deck, eclState);
```

Comment:
true, i'm gonna fix this. but didn't you say that gas was enabled? (or did "No gas is specified." mean "Gas is disabled" instead of "No, gas is specified."?)